app/serializers/api/contact_form_item_serializer.rb

```diff
@@ -0,0 +1,11 @@
+class Api::ContactFormItemSerializer < ActiveModel::Serializer
+  attribute(:corners) { enum_to_hash(object.corners) }
+  attribute(:departments) { enum_to_hash(object.departments) }
+  attribute(:grades) { enum_to_hash(object.grades) }
+
+  private
+
+    def enum_to_hash(enum)
+      Hash[*enum.map { |e| [e.t, e.value] if e.t != '未記入' }.compact.flatten]
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.


<!-- Comment by [SideCI](https://sideci.com) -->